pyproject.toml

```diff
@@ -23,30 +110,91 @@ known_cpg = [
-
-fixable = ["A", "B", "C", "D", "E", "F", "G", "I", "N", "Q", "S", "T", "W", "ANN", "ARG", "BLE", "COM", "DJ", "DTZ", "ERA", "EXE", "FBT", "ICN", "ISC", "NPY", "PD", "PGH", "PIE", "PL", "PT", "PYI", "RET", "RSE", "RUF", "SIM", "SLF", "TCH", "TID", "UP", "YTT"]
+# select = ["A", "B", "C",  "E", "F", "G", "I", "N", "Q", "S", "W", "ANN", "ARG", "BLE", "COM", "DJ", "DTZ", "ERA", "EXE", "ICN", "ISC", "NPY", "PD", "PGH", "PIE", "PL", "PT", "PYI", "RET", "RSE", "RUF", "SIM", "SLF", "TCH", "TID", "UP", "YTT"]
+# fixable = ["A", "B", "C", "D", "E", "F", "G", "I", "N", "Q", "S", "T", "W", "ANN", "ARG", "BLE", "COM", "DJ", "DTZ", "ERA", "EXE", "FBT", "ICN", "ISC", "NPY", "PD", "PGH", "PIE", "PL", "PT", "PYI", "RET", "RSE", "RUF", "SIM", "SLF", "TCH", "TID", "UP", "YTT"]
 
 ignore = [
-    "ANN101",  # Missing type annotation for self in method
-    "ANN201",  # Missing return type annotation for public function
-    "E501",    # Line length too long
-    "E731",    # Do not assign a lambda expression, use a def
-    "E741",    # Ambiguous variable name
-    "G004",    # Logging statement uses f-string
-    "PLR0911", # Too many return statements
-    "PLR0912", # Too many branches
-    "PLR0913", # Too many arguments to function call
-    "PLR0915", # Too many statements
-    "PLW0603", # Using the global statement to update `<VAR>` is discouraged
-    "PT018",   # Assertion should be broken down into multiple parts
-    "Q000",    # Single quotes found but double quotes preferred
-    "S101",    # Use of assert detected
-    "I001", # Import block is un-sorted (we use isort directly as ruff sort isn't fully supported)
+    # "ANN101",  # Missing type annotation for self in method
+    # "ANN201",  # Missing return type annotation for public function
+    # "E501",    # Line length too long
+    # "E731",    # Do not assign a lambda expression, use a def
+    # "E741",    # Ambiguous variable name
+    # "G004",    # Logging statement uses f-string
+    # "PLR0911", # Too many return statements
+    # "PLR0912", # Too many branches
+    # "PLR0913", # Too many arguments to function call
+    # "PLR0915", # Too many statements
+    # "PLW0603", # Using the global statement to update `<VAR>` is discouraged
+    # "PT018",   # Assertion should be broken down into multiple parts
+    # "Q000",    # Single quotes found but double quotes preferred
+    # "S101",    # Use of assert detected
+    # "I001", # Import block is un-sorted (we use isort directly as ruff sort isn"t fully supported)
 ]
+
+[tool.semantic_release]
+version_variables = ["src/cpg_flow/__init__.py:__version__", "pyproject.toml:project.version"]
+assets = []
+build_command_env = []
+commit_message = "{version}\n\nAutomatically generated by python-semantic-release"
+commit_parser = "angular"
+logging_use_named_masks = false
+major_on_zero = true
+allow_zero_version = true
+repo_dir = "/Users/sabyan/Desktop/cpg-flow"
```

Comment:
Could we change this to a relative path instead of an absolute path?

Reply:
Done